### ISSUE TITLE
Composer fails to install latest version - only the second-newest works

### ISSUE THREAD

**Issue by kabachello** [hiqdev/asset-packagist#45]
For the following composer.json will install paste.js version 0.0.20, although 0.0.21 is available. 
```
{
    "require": {
        "bower-asset/paste.js": "^0.0.20"
    },
    "repositories": {
        "asset-packagist": {
            "type": "composer",
            "url": "https://asset-packagist.org"
        }
	}
}
```
Requireing ^0.0.21 will to the following error:  "The requested package bower-asset/paste.js ^0.0.21 exists as bower-asset/paste.js[0.0.21, 0.0.20, 0.0.19, 0.0.18, 0.0.16, 0.0.15, 0.0.14, 0.0.13, 0.0.12,0.0.11, 0.0.10, 0.0.9, 0.0.8, 0.0.7, 0.0.6, 0.0.5, 0.0.4, 0.0.3, 0.0.2, 0.0.1, dev-master] but these are rejected by your constraint."

Setting the version to ">=0.0.21" will yield "The requested package bower-asset/paste.js >=0.0.21 is satisfiable by bower-asset/paste.js[dev-master] but these conflict with your requirements or minimum-stability."

I'm pretty sure, it was OK earlier. I'm using composer 1.4.1 and PHP 7 here.

**Comment by hiqsol** [comment]
Fixed version 0.0.21 to work.

Please pay attention that version constraint "^0.0.20" will NOT allow installation of 0.0.21 because it is pre-1.0
https://getcomposer.org/doc/articles/versions.md#caret

**Comment by kabachello** [comment]
Wow! That was fast. It's working now - thanks!
